app/styles.css

```diff
@@ -83,31 +83,31 @@ h1, h2, h3, h4, h5, h6 {
   font-size: 40px;
 }
 
-#cards div:nth-child(7n+1) a{
+#cards div:nth-child(7n+1) .btn-group a{
```

Comment:
By doing this you're breaking the functionality of the rainbow on the front page. This is because the ones on the home page aren't inside a `.btn-group`.  If you remove the `.btn-group` it should still select all the `a` elements.